docs/articles/index.md

```diff
@@ -0,0 +1,136 @@
+# Index
+
+[A](#a) · [B](#b) · [C](#c) · [D](#d) · [E](#e) · [F](#f) · [G](#g) · [H](#h) · [I](#i) · [J](#j) · [K](#k) · [L](#l) · [M](#m) · [N](#n) · [O](#o) · [P](#p) · [Q](#q) · [R](#r) · [S](#s) · [T](#t) · [U](#u) · [V](#v) · [W](#w) · [X](#x) · [Y](#y) · [Z](#z)
+
+---
+
+## A
+[Access Control](access-control.md)  
+[ACLs](access-control.md)  
+[API](api.md)  
+[Application Groups](application-groups.md)  
+[Applications](applications.md)  
+[Artifacts](artifacts.md)  
+[Authentication](access-control.md)  
+[Authorization](access-control.md)  
+
+## B
+[Backoff Strategy](backoff-strategy.md)  
+[Bridge Networking](networking.md#bridge-networking)
+
+## C
+[CLI](cli.md)  
+[Cluster Maintenance](cluster-maintenance.md)  
+[Command Line Arguments](command-line-arguments.md)  
+[Configuration](configuration.md)  
+[Constraints](constraints.md)  
+[Containers](containers.md)  
+[Custom Executor](executors.md)  
+
+## D
+[Debugging](debugging.md)  
+[Dependencies](dependencies.md)  
+[Deployments](deployments.md)  
+[Docker](docker.md)  
+[Download](download.md)  
+[Downscaling](downscaling.md)  
+
+## E
+[Endpoints](endpoints.md)  
+[Env Var Based Secrets](secrets.md)  
+[Environment Variables](environment-variables.md)  
+[Ephemeral Volumes](ephemeral-volumes.md)  
+[Event-bus](event-bus.md)  
+[Events](events.md)  
+[Executors](executors.md)  
+[Exponential Backoff](backoff-strategy.md)  
+[External Volumes](external-volumes.md)  
+[Features](features.md)  
+
+## F
+[File Based Secrets](secrets.md)  
+
+## H
+[Health Checks](health-checks.md)  
+[Host Networking](host-networking.md)  
+[HTTP API](api.md)  
```

Comment:
`Rest API` instead of `HTTP API`?

Reply:
There was contention in the past over whether our API is Restful or not. Both is fine for me for the sake of the article name.

Other opinions? I'm happy to rename.